apps/finance/contracts/Finance.sol

```diff
@@ -119,6 +119,7 @@ contract Finance is AragonApp {
         initialized();
 
         require(_periodDuration > 1);
+        require(_vault != address(0));
```

Comment:
Maybe we could even check that `isContract`.

Reply:
Was thinking the same, but even with `isContract` we can't be sure that it's a Vault... so not sure it's worth the extra gas. We're also not checking that in a few places (e.g. for the kernel in the app proxies).

Reply:
Well, I think the same reasoning would apply to several other places where we are using `isContract` and that even not making sure it is a Vault, at least we could avoid some more errors than only checking 0 address (so we increase safety although not up to 100%), and about the gas this function is called only once, but I'm fine leaving it this way.

Reply:
Thinking about it some more, safer is better :). Will wait until `IsContract` is released in a new aragonOS.